src/Common/tests/System/Diagnostics/AssertWithCallerAttributes.cs

```diff
@@ -115,6 +115,20 @@ internal static class Assert
         catch (Exception e) { throw WrapException(e, path, line); }
     }
 
+    public static void DoesNotContain(IEnumerable collection, object expected,
+        [CallerFilePath] string path = null, [CallerLineNumber] int line = 0)
+    {
+        try { Xunit.Assert.DoesNotContain(expected, (IEnumerable<object>)collection); }
+        catch (Exception e) { throw WrapException(e, path, line); }
+    }
+
+    public static void DoesNotContain<T>(IEnumerable<T> collection, T expected,
```

Comment:
The purpose of these wrappers is to have the exact same signature as the xunit ones so they can be drop-in replacements.  The order of the arguments should be the same, there should be a one-to-one correspondence between the wrapper methods and the xunit methods being called, etc.